.chromatic/custom-addons/chromatic/index.js

```diff
@@ -1,7 +1,10 @@
-import {expressThemes, locales, scales, themes} from '../../constants';
+import {expressThemes, locales, S2Backgrounds, S2ColorThemes, scales, themes} from '../../constants';
 import {makeDecorator} from '@storybook/preview-api';
 import {Provider, View} from '@adobe/react-spectrum';
+import {Provider as S2Provider} from '@react-spectrum/s2';
 import React, {useEffect} from 'react';
+// TODO: can't seem to use the style macro here...
+// import {style} from '../../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
```

Comment:
Not exactly sure why the macro doesn't work here, ideally I'd use if for some of the divs below

Reply:
can't import directly from `@react-spectrum/s2` either? instead of the relative path?

Reply:
yep, doing that returns an error of 
```
@parcel/transformer-js: Error loading macro: Could not resolve module "@react-spectrum/s2/style" from 
"/Users/danilu/dev/react-spectrum/.chromatic/custom-addons/chromatic/index.js"
```

 The relative path way returns an error of 
```
@parcel/core: Failed to resolve '0' from './.chromatic/custom-addons/chromatic/index.js'
@parcel/resolver-default: Cannot find module '0'
```
I haven't had time to dig into too deeply but its not a huge deal IMO since it is just some padding/div wrappers for story layout

Reply:
o, maybe this is loaded through another bundler, storybook UI has one vs the one used for stories